appveyor.yml

```diff
@@ -24,7 +24,10 @@ build_script:
   - conda update conda
   - conda install conda-build
   - ps: if(-not $env:APPVEYOR_PULL_REQUEST_NUMBER) { conda install anaconda-client }
-  - conda build conda.recipe --channel dynd/channel/dev
+  - conda build conda\ndt.recipe --channel dynd/channel/dev
+  - rd /q /s build dist dynd.ndt.egg-info
+  - echo dynd.ndt >> requirements.txt
```

Comment:
Why is this necessary? Also, this line doesn't do what you think it does. `echo` takes precedence over the redirect. Luckily the redirect doesn't have to be put at the end of the line, so `>> requirements.txt echo dynd.ndt` should do what you want.